{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""{{ cookiecutter.project_slug }}'s main module."""
+
+__version__ = "1.0.0"
```

Comment:
Possibly the initial version should not be a release version.

Reply:
OK, fair, revert to `0.1.0` in https://github.com/Leibniz-HBI/smormlpy/pull/20/commits/2c3d7134352e734ae08a2209826cfb3ce799bd31